README.md

```diff
@@ -135,22 +135,20 @@ You can read about the formatting guide [here](https://github.com/space-concordi
 For a quick primer on our workflow using git, [CLICK HERE :)](https://github.com/space-concordia-robotics/robotics-prototype/wiki/Git-Workflow-and-Conventions)
 ### Commit message hook
```

Comment:
I think changing this title to simply "Git hooks" is more appropriate now that we have more than one type.